### DISCUSSION TITLE
Workflow for Projects with Bazel

### DISCUSSION THREAD

**Discussion by davidzchen** [community/community#114306] [Actions]
### Select Topic Area

Product Feedback

### Body

Currently, there are workflows for projects built with a number of build systems, such as CMake, MSBuild, Make, etc. There are many widely-used projects built with Bazel, but there does not seem to be a workflow for Bazel projects.

Would it be possible to implement a GitHub Actions workflow for projects built with Bazel, similar to these other workflows?

**Comment by ADhd-a** [comment]
[Uploading docs-main.zip…]()
